wikiclue/src/store/gameplay.ts

```diff
@@ -0,0 +1,25 @@
+import { writable } from 'svelte/store';
+
+// Levels Variables
+const levelsStreakCount = writable(0);
+const levelsTimeRemaining = writable(60);
+const levelsSkipsRemaining = writable(3);
```

Comment:
Levels won't have skips, streaks, or time remaining to my understanding. I thought it would just be untimed, unskippable levels in order?